amoro-core/src/main/java/org/apache/amoro/process/OptimizingStage.java

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-package org.apache.amoro.api.process;
+package org.apache.amoro.process;
```

Comment:
What is the reason for moving these 3 classes from module `amoro-ams-api` to module `amoro-core`?

Reply:
TableDescriptor depends on those class, and the TableDescriptor was moved to amoro-core

Reply:
Shall we move other classes into module `amoro-core` too in the next PR?
If yes, this make sense to me.